### DISCUSSION TITLE
GitHub pages redirect to old domain, but new works

### DISCUSSION THREAD

**Discussion by meteoritt** [community/community#22248] [Other Feature Feedback, Questions, & Ideas]
<p>I have problem with update GitHub Pages custom domain.</p>
<p>Repository: <a href="https://github.com/CSRedRat/csredrat.github.io" rel="noopener nofollow ugc">https://github.com/CSRedRat/csredrat.github.io</a><br>
New custom domain changed from repo settings, displayed actual domain in input field and published address, latest CNAME file, but <a href="http://csredrat.github.io" rel="noopener nofollow ugc">csredrat.github.io</a> redirect to old custom domain.</p>
<p>UPDATED: problem resolved, need wait or clear cache.</p>

**Comment by meteoritt** [comment]
<p>(post withdrawn by author, will be automatically deleted in 24 hours unless flagged)</p>

**Comment by darkRaspberry** [comment]
<p>Its too much confusing. Can you clear things a little bit? I can help. It look like you are using more than one github pages in your account. Its look like its working fine.</p>

**Comment by meteoritt** [comment]
<p>(post withdrawn by author, will be automatically deleted in 24 hours unless flagged)</p>

**Comment by meteoritt** [comment]
<p>Strange, but <a href="https://csredrat.github.io/topalias/" rel="noopener nofollow ugc">https://csredrat.github.io/topalias/</a> correct redirect to <a href="http://csredrat.gq/topalias/" rel="noopener nofollow ugc">http://csredrat.gq/topalias/</a></p>

**Comment by darkRaspberry** [comment, marked as answer]
<aside class="quote no-group" data-username="CSRedRat" data-post="2" data-topic="144796">
<div class="title">
<div class="quote-controls"></div>
 CSRedRat:</div>
<blockquote>
<p>GitHub Pages domain <a href="http://csredrat.github.io" rel="noopener nofollow ugc">csredrat.github.io</a> redirected to csredrat.ml, but must to csredrat.gq</p>
</blockquote>
</aside>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/b/1bfcc653f47dbafd2be3c7b3e2f61bbcdceb81e2.jpeg" data-download-href="/uploads/short-url/3ZAssxPCZERdkdRz4kCT6tAvsbw.jpeg?dl=1" title="Screenshot from 2020-11-22 17-49-57" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/56693429/181069723-f0c1d8b7-05e8-4195-a405-1a159d7b791a.jpeg" alt="Screenshot from 2020-11-22 17-49-57" data-base62-sha1="3ZAssxPCZERdkdRz4kCT6tAvsbw" width="690" height="387" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/b/1bfcc653f47dbafd2be3c7b3e2f61bbcdceb81e2_2_690x387.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/1/b/1bfcc653f47dbafd2be3c7b3e2f61bbcdceb81e2_2_1035x580.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/1/b/1bfcc653f47dbafd2be3c7b3e2f61bbcdceb81e2.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/b/1bfcc653f47dbafd2be3c7b3e2f61bbcdceb81e2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot from 2020-11-22 17-49-57</span><span class="informations">1366×768 122 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Nope it redirect to csredrat.gq<br>
You have problem with <a href="https://en.wikipedia.org/wiki/Time_to_live" rel="noopener nofollow ugc">TTL</a> just wait or use incognito or clear your cache.</p>
